### ISSUE TITLE
Issue ID: DAC_Actionable_Element2

### ISSUE THREAD

**Issue by VDuczekW** [guardian/dotcom-rendering#5034]
Full report: https://drive.google.com/file/d/1ie4-q1R97_rF8I-AvLcR9SvpECITt1BX/view?usp=sharing

Full issue list: https://docs.google.com/spreadsheets/d/1vWQtZfD49uq7uNCNELuYN_HcIj-BKiE4t4Hwda3tJaQ/edit#gid=0

**Comment by bryophyta** [comment]
>**Note**
>This is an automatically-generated comment. Feel free to edit the content, but please don't edit this 'note' block.
 
# Intro to report

Copying the intro here to aid discoverability of the report's content:
```
Issue ID: DAC_Actionable_Element2
URL: https://www.theguardian.com/uk
Page title: News, sport and opinion from the Guardian's UK edition | The Guardian
Journey: 1.1
Screenshot:

Users can interact with the play button found within the background image leading to a
video at the bottom of the page by using a mouse; however, this button cannot be actioned
using keyboard only commands or be referenced using voice activation software. This limits
how users can navigate through the site and can potentially impact on whether they are
able to watch the video contained, which puts them at a disadvantage to other users.
```

**Comment by jamesgorrie** [comment]
@HarryFischer - do we have any update on what we are doing with video carousels? This should be taken into account when building that.

It would be good to know if we are going to deprecate this one before we start any work on it.

**Comment by HarryFischer** [comment]
@jamesgorrie Video carousels should be built in DCR. Tweaked design is here: https://www.figma.com/file/5CfbWOeZPRi15lXBD5u1rW/Card-layout-system?node-id=218%3A4449

Bella and I spoke with DAC about this the other day though so there will be some a11y improvements to make

**Comment by OllysCoding** [comment]
We can switch the element to `button` here: https://github.com/guardian/frontend/blob/a1fbdd28b5f37c3446be5376a575a8ca8ac0274b/common/app/views/fragments/atoms/youtube.scala.html#L90-L94

We should also ensure the new one built in DCR is accessible

**Comment by bryophyta** [comment]
> do we have any update on what we are doing with video carousels? This should be taken into account when building that.
> It would be good to know if we are going to deprecate this one before we start any work on it.

There's now an issue for the carousel a11y work: #5975. Although as far as I'm aware there's still some work being done to finalise the spec for this?

For the particular issue highlighted in the DAC report on _this_ issue: am I right in thinking that @OllysCoding's suggested fix in the comment above will address this issue, and can be done independent of any work on the carousel itself?
